include/sys/poll.h

```diff
@@ -78,8 +78,7 @@
 
 #define POLLFD       (0x00)
 #define POLLFILE     (0x40)
-#define POLLSOCK     (0x80)
-#define POLLMASK     (0xC0)
+#define POLLMASK     (0x40)
```

Comment:
I think it is better if `POLLMASK` is defined as an OR operation of all previously defined event types.
```suggestion
#define POLLMASK     (POLLFD | POLLFILE)
```

Reply:
keep for consistency?

Reply:
Sorry, I don't understand how the proposed approach is less consistent than the current one.
In fact, I think it is kinda normal to define a mask based on the existing values the it should "mask".

Reply:
okay, done!